pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1194,7 +1194,20 @@ func lookupDomainConfig(ctx *domainContext, key string) *types.DomainConfig {
 	return &config
 }
 
+func runningOnKVM() bool {
```

Comment:
Looks good in general but maybe this is a good time to extend https://github.com/lf-edge/eve/blob/5508ecb0dc9d2e619634bc89b3a70bd95d1ea7c9/pkg/edgeview/vendor/github.com/lf-edge/eve/pkg/pillar/types/domainmgrtypes.go#L525 to have a CpuSet boolean?

(And also a UseVhost boolean if we also want to turn the explicit check in volumemgr to calling hypervisor.GetCapabilities()).

Reply:
Maybe.
Am I right it also involves changes in the EVE API (`info.pb.go`)?

Reply:
Also, if we extend the Capabilities structure and want to use it also in Volume Manager, would it be better to move the structure to another package, common for both Domain Manager and Volume Manager?

Reply:
> Maybe. Am I right it also involves changes in the EVE API (`info.pb.go`)?

No impact on that API. This is just a local API to the hypervisor package.

> Also, if we extend the Capabilities structure and want to use it also in Volume Manager, would it be better to move the structure to another package, common for both Domain Manager and Volume Manager?

Technically it belongs to the hypervisor package, so no problem moving the capabilities struct to that package. (I suspect it was accidental that the struct was defined in the types package).

Reply:
Ok, I got it. I will not touch the API then.
Actually, the `domainmgrtypes.go` package is a shared one, so it's not a problem, that the structure belongs there.

Reply:
done